servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpContextKeys.java

```diff
@@ -68,6 +68,9 @@ public final class HttpContextKeys {
     public static final Key<Boolean> HTTP_FORCE_NEW_CONNECTION =
             newKey("HTTP_FORCE_NEW_CONNECTION", Boolean.class);
 
+    public static final Key<Boolean> HTTP_OPTIMIZE_ERROR_STREAM =
```

Comment:
1. This is an internal key, not intended for use by others. It's unfortunate that we need to set it in grpc-api and read in grpc-netty. Instead of creating a public key, consider creating a new class in `servicetalk-grpc-internal` module, something like `GrpcContextKeys`.

2. To align with gRPC protocol wording, consider naming it `TRAILERS_ONLY_RESPONSE`.

3. All new public API should have javadoc, even in internally public classes. Looks like this checkstyle rule is broken now, but adding javadoc now will simplify our live when we fix checkstyle. It will be helpful for us as well to have a note why we need this key.